lib/plugins/extension/script.js

```diff
@@ -110,4 +110,23 @@ jQuery(function(){
         );
     });
 
-});
\ No newline at end of file
+    /**
+        Create section for enabling/disabling viewing options
+     */
+    if ( $extmgr.find('.tabs li:first-child, .tabs li:nth-child(2)').hasClass('active') ) {
```

Comment:
Might be better to identify tabs by class names instead of relying on their order (which might change in future versions)